extensions/brew/src/brew.ts

```diff
@@ -116,7 +118,7 @@ export const brewPrefix = (() => {
 })();
 
 export const brewPath = (suffix: string) => path_join(brewPrefix, suffix);
-export const brewExecutable = () => brewPath("bin/brew");
+export const brewExecutable = () => preferences.customBrewPath && preferences.customBrewPath.length > 0 ? preferences.customBrewPath : brewPath("bin/brew");
```

Comment:
Thanks for spotting the omission here 😄

Reply:
Actually, now I think about it, this change is required. Or at least it should not be if the custom brew path is set correctly since `brewPath("bin/brew") == customBrewPath`.